### ISSUE TITLE
Reconnect still fails :(

### ISSUE THREAD

**Issue by MTRNord** [Nordgedanken/matrix-twitch-bridge#6]

**Comment by MTRNord** [comment]
Fixed in 5605fd2d96cc2355b6913b91c499e5f382abb2f4

**Comment by MTRNord** [comment]
Current errror log:

```
[Apr 19, 2018 21:35:45] [ERROR] websocket: close 1006 (abnormal closure): unexpected EOF
[Apr 19, 2018 21:35:45] [WARN] Done got closed
[Apr 19, 2018 21:35:45] [WARN] Closing old WS
[Apr 19, 2018 21:35:45] [ERROR] write tcp 10.0.2.15:41798->35.164.86.102:443: i/o timeout
```
